package-res/reportviewer/reportviewer.js

```diff
@@ -951,16 +954,20 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
 
                     dlg.hide();
                   } else {
-                    var urlClean = url.substring(0, url.indexOf("/api/repos")) + '/plugin/reporting/api/jobs/' + this._currentReportUuid  + '/clean';
```

Comment:
Can you remove that "/clean" call. It will trigger the race otherwise, as /clean removes the future, and the /content call can then not find the content on the server.